test/controllers/data_exports_controller_test.rb

```diff
@@ -275,19 +275,20 @@ class DataExportsControllerTest < ActionDispatch::IntegrationTest
   end
 
   test 'should redirect from project PUID' do
-    get redirect_from_data_export_path(@data_export1, puid: 'INXT_PRJ_AAAAAAAAAA')
+    get redirect_data_export_path(@data_export1, identifier: 'INXT_PRJ_AAAAAAAAAA')
     assert_response :redirect
   end
 
   test 'should redirect from sample PUID' do
-    get redirect_from_data_export_path(@data_export1, puid: 'INXT_SAM_AAAAAAAAAA')
+    get redirect_data_export_path(@data_export1, identifier: 'INXT_SAM_AAAAAAAAAA')
     assert_response :redirect
   end
 
   test 'should list samples' do
     post list_data_exports_path(format: :turbo_stream), params: {
       page: 1,
-      sample_ids: [@sample1.id]
+      sample_ids: [@sample1.id],
+      list_class: 'sample'
```

Comment:
Is it worth testing the workflow_execution list_class get tested too?

Reply:
Added in f781876c5c17b14660db3619e035a4ff5bb3d3ec